### PR TITLE
release React Native demo application

### PR DESCRIPTION
**Pull request by spencerlepine**
#  🔴 BLOCKED:  `yarn add amazon-connect-chatjs@1.5.0`

*Issue #, if available:*
Resolves https://github.com/amazon-connect/amazon-connect-chatjs/issues/42
Addresses https://github.com/amazon-connect/amazon-connect-chat-ui-examples/issues/99

*Description of changes:*

Releasing a React Native + ChatJS demo application. Must use latest version: `amazon-connect-chatjs@1.5.0`

### Checklist
 - [x] Create `connectReactNativeChat` UI-example
 - [x] Create CI workflow for `connectReactNativeChat`
 - [x] Update README
 - [x] Update CHANGELOG
 - [ ] Create release in this repo

### Testing
 - ✅ Passed unit tests - https://github.com/spencerlepine/amazon-connect-chat-ui-examples/actions/runs/4556126880
 - ✅ Local testing (see screenshot)

Screenshot 2023-03-29 at 10 33 01 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: connectReactNativeChat/src/components/ChatSession.js [hunk 1]
```diff
@@ -0,0 +1,838 @@
...
+// Low-level abstraction on top of Chat.JS
+class ChatJSClient {
+  session = null;
+
+  constructor(chatDetails, region, customNetworkStatus) {
+    if (
+      ENABLE_CHATJS_LOGS &&
+      connect.ChatSession.setGlobalConfig &&
+      typeof connect.ChatSession.setGlobalConfig === "function"
+    ) {
```
Comment (spencerlepine) [inline connectReactNativeChat/src/components/ChatSession.js:36]: Must clean this up

File: connectReactNativeChat/src/components/ChatWrapper.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+import ChatSession from "../components/ChatSession";
+import initiateChat from "../api/initiateChat";
+import { startChatRequestInput } from "../../config";
+import filterIncomingMessages from "../utils/filterIncomingMessages";
+import NetInfo from "@react-native-community/netinfo";
+
+let isOnline = true;
+const customIsNetworkOnline = () => {
+  console.log("Checking if online:", isOnline);
```
Comment (spencerlepine) [inline connectReactNativeChat/src/components/ChatWrapper.js:15]: clean this up -  few top lines

File: connectReactNativeChat/src/utils/convertConnectToGiftedChatMsg.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+import {
+  CUSTOMER_USER,
+  AGENT_USER,
+  supportMessageContentTypes,
+} from "../../config";
+
+const isValidConnectChatMessage = (msg) => {
+  return (
+    msg.id &&
```
Comment (spencerlepine) [inline connectReactNativeChat/src/utils/convertConnectToGiftedChatMsg.js:12]: clean this up, impossible to read

File: connectReactNativeChat/src/components/ChatWrapper.js [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+      })
+    } else {
+      openHomeScreen()
+    }
+  }
+
+  const handleSendMessage = (msg) => {
+    const content = {
+      data: msg,
+      type: 'text/plain',
```
Comment (haomingli2020) [inline connectReactNativeChat/src/components/ChatWrapper.js:105]: Does this mean all messages will be text/plain and users can not send markdown?

**Pull request closed without merging**